### PR TITLE
add unassign method

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8430

"specialized policy" term taken from https://docs.microsoft.com/en-us/microsoftteams/policy-assignment-overview

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/assign-policies-users-and-groups.md [hunk 1]
```diff
@@ -32,13 +32,16 @@ Follow these steps to assign a policy to an individual user or to a small number
...
-1. In the left navigation of the Microsoft Teams admin center, go to **Users**, and then select the user.
+1. In the left navigation of the Microsoft Teams admin center, click to expand **Users**, and then select **Manage users**.
 2. Select the user by clicking to the left of the user name, and then select **Edit settings**.
 3. Select the policy you want to assign, and then select **Apply**.
 
 ![Assign a policy to a user in the Teams admin center.](media/assign-policy-user.png)
 
-Or, you can also do the following:
+> [!NOTE]
+> To unassign a specialized policy from a user you can select Global (Org-wide default) for each policy type.
```
Comment (JohanFreelancer9) [inline Teams/assign-policies-users-and-groups.md:42]:
```suggestion
> To unassign a specialized policy from a user you can select **Global (Org-wide default)** for each policy type.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion, please.

**Comment by get-itips** [comment]
@tiburd Can you help me getting this merged ? Thank you

**Pull request merged**